server/src/main/java/org/elasticsearch/action/fieldcaps/FieldCapabilitiesNodeRequest.java

```diff
@@ -143,12 +143,16 @@ public ActionRequestValidationException validate() {
     public String getDescription() {
         final StringBuilder stringBuilder = new StringBuilder("shards[");
         Strings.collectionToDelimitedStringWithLimit(shardIds, ",", "", "", 1024, stringBuilder);
+        return completeDescription(stringBuilder, fields, filters, allowedTypes);
```

Comment:
I optimised the string building here a little and dried this up. Not sure this ever gets hot but no need to burn more heap than necessary when building this large string.